### PR TITLE
Conform token select

### LINKED ISSUE #1262: Conform Select tokens

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1262

### CODE REVIEW

File: libraries/core-react/src/components/Select/NativeSelect/Select/Select.tokens.ts [hunk 1]
```diff
@@ -1,67 +1,55 @@
...
     icon: {
       width: '24px',
     },
-    border: {
-      outline: { color: 'transparent', width: '1px' },
-      bottom: { color: colors.text.static_icons__tertiary.hex, width: '1px' },
+  },
+  border: {
+    type: 'bordergroup',
+    // outline: { color: 'transparent', width: '1px' },
```
Comment (pomfrida) [inline libraries/core-react/src/components/Select/NativeSelect/Select/Select.tokens.ts:33]: Oops I missed this comment last time sorry 😓

**Review by pomfrida — CHANGES_REQUESTED** [review]
Just a comment

File: libraries/core-react/src/components/Select/NativeSelect/Select/Select.tokens.ts (hunk 1 above)
Comment (pomfrida) [inline libraries/core-react/src/components/Select/NativeSelect/Select/Select.tokens.ts:33]: Comment

Comment (mimarz) [reply libraries/core-react/src/components/Select/NativeSelect/Select/Select.tokens.ts]: Fixed

**Review by pomfrida — APPROVED** [review]
LGTM 👍

**Pull request merged**